fuzzing/src/utils/FreeTypeStream.h

```diff
@@ -18,18 +18,17 @@
 
 #include <cstdint>
 
-#include <boost/core/noncopyable.hpp>
-
 #include <ft2build.h>
 #include FT_FREETYPE_H
 #include <freetype/internal/ftstream.h>
 
+#include "utils/noncopyable.h"
 
 namespace freetype {
```

Comment:
💄 let's have two consecutive empty lines between the last `#include` + the first `namespace` please

Reply:
Done.